tests/baselines/reference/user/bluebird.log

```diff
@@ -1,6 +1,8 @@
 Exit Code: 1
 Standard output:
 node_modules/bluebird/js/release/assert.js(11,30): error TS2339: Property 'constructor$' does not exist on type 'Error'.
+node_modules/bluebird/js/release/async.js(38,14): error TS2339: Property 'hasDevTools' does not exist on type 'typeof ret'.
+node_modules/bluebird/js/release/async.js(91,11): error TS2339: Property 'hasDevTools' does not exist on type 'typeof ret'.
```

Comment:
`ret` is the name of the exported object from bluebird's utils.js, which is first assigned and then conditionally added to property-assignment:

```js
var ret = {
  isNode: isNode,
  isClass: isClass, // etc
}
ret.isRecentNode = ret.isNode && (function() { ... })();
```

Previously, `isNode` and `isRecentNode` were both bound, but now only `isRecentNode` is. This is a regression.

Reply:
bisecting points at the globalThis PR. I'll take a look. #30159